superset/commands/dataset/update.py

```diff
@@ -86,15 +86,21 @@ def validate(self) -> None:
         except SupersetSecurityException as ex:
             raise DatasetForbiddenError() from ex
 
-        database_id = self._properties.get("database", None)
```

Comment:
The default for `dict.get()` is `None` and thus there's no need to explicitly define it.